README.rst

```diff
@@ -834,6 +834,12 @@ Install test requirements:
 
     pip install -r requirements-test.txt
 
+Set up the pre-push hook to run tests automatically:
```

Comment:
I'd make it more precise:

Set up the pre-push hook to run tests and QA checks automatically right before the git push action, so that if anything fails the push operation will be aborted.